mesa/experimental/jupyter_viz.py

```diff
@@ -9,48 +8,72 @@
 
 import mesa
 
-# Avoid interactive backend
-plt.switch_backend("agg")
```

Comment:
This was added so that user code that does the pyplot interface (`plt.plot`, etc) can still work, even though not recommended.

Reply:
ah, makes sense. I reverted the change